mfs/system.go

```diff
@@ -1,25 +1,22 @@
-// package ipnsfs implements an in memory model of a mutable ipns filesystem,
+// package mfs implements an in memory model of a mutable ipns filesystem,
 // to be used by the fuse filesystem.
 //
 // It consists of four main structs:
 // 1) The Filesystem
 //        The filesystem serves as a container and entry point for the ipns filesystem
-// 2) KeyRoots
-//        KeyRoots represent the root of the keyspace controlled by a given keypair
+// 2) Root
+//        Root represent the root of the keyspace controlled by a given keypair
```

Comment:
This comment should not say "the root of the keyspace controlled by a given keypair" any more, right?